west.yml

```diff
@@ -68,7 +68,7 @@ manifest:
       revision: b4d31f33238713a568e23618845702fadd67386f
       path: modules/hal/nuvoton
     - name: hal_nxp
-      revision: 9a52c50338dcec8ab938f91f62e829898e298d14
+      revision: refs/pull/92/head
```

Comment:
The HAL PR has now been merged, please update this to the new SHA.

Reply:
Updated